test/integration/indexer/migration_test.rb

```diff
@@ -1,9 +1,15 @@
 require 'integration_test_helper'
 
+# FIXME: this tests the BulkLoader. We don't think we need this anymore.
+# The "Migration" in the name means creating new indexes and copying data from
+# the existing ones.
```

Comment:
We need to actually switch this over in the search_analytics codebase which I haven't got to yet.